packages/e2e-tests/specs/editor/various/block-switcher.test.js

```diff
@@ -82,9 +82,9 @@ describe( 'Block Switcher', () => {
 		await pressKeyWithModifier( 'alt', 'F10' );
 
 		// Verify the block switcher exists.
-		expect( await hasBlockSwitcher() ).toBeTruthy();
+		expect( await hasBlockSwitcher() ).toBeFalsy();
 		// Verify the correct block transforms appear.
-		expect( await getAvailableBlockTransforms() ).toHaveLength( 1 );
+		expect( await getAvailableBlockTransforms() ).toHaveLength( 0 );
```

Comment:
The comments in this test no longer seem to agree with the code. Where is the change coming from?

Reply:
The comments should actually make sense because I'm merely reverting
https://github.com/WordPress/gutenberg/pull/43181/files#diff-56243bd4e7ab153498f10d75b0827121efa47fe088f53d75332a093e91a7edf2